content/en/docs/ct-logs.html

```diff
@@ -67,6 +63,24 @@ <h2>CT Logs</h2>
 Information about the various lifecycle states that a CT log progresses through can be found <a href="https://googlechrome.github.io/CertificateTransparency/log_states.html">here</a>.
 </p>
 
+<h2>Sunlight</h2>
```

Comment:
By virtue of making this an H2, the "Production" and "Testing" H3s are now sub-sections of "Sunlight", rather than being subsections of "CT Logs". Either this should be an H3, or the "CT Logs" section should be moved below this and maybe renamed to "RFC 6962 Logs".

Reply:
Switched to H3, which matches where I'd intended to put it hierarchically.